notify/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:6.9.2-alpine
+COPY ./app/config.json /root/
+COPY ./app/ /var/www/html/web/
+COPY ./src/wait /root/
+RUN chmod +x /root/start.sh
+RUN chmod +x /root/run.sh
```

Comment:
Doing `chmod` inside of Dockerfile looks wired. Instead I recommend to chmod files in git repository